README.md

```diff
@@ -1,2 +1,178 @@
-# test-file-suffix-inspector
-GPT Sure, here's a concise description for the test-file-suffix-inspector repository:  Repository Name: test-file-suffix-inspector  Description: The Test File Suffix Inspector GitHub Action scans test files in src/test directories to ensure naming conventions, reporting any files missing specified suffixes.
+# Test File Suffix Inspector
+
+- [Description](#description)
+- [How It Works](#how-it-works)
+- [Inputs](#inputs)
+- [Outputs](#outputs)
+- [Usage Example](#usage-example)
+  - [Default](#default)
+  - [Full example](#full-example)
+- [How To Build](#how-to-build)
+- [Running Unit Tests](#running-unit-tests)
+- [Code Coverage](#code-coverage)
+- [Run Action Locally](#run-action-locally)
+- [Contributing](#contributing)
+- [License](#license)
+
+## Description
+The **Test File Suffix Inspector** GitHub Action is designed to ensure naming conventions in test files within specified repository directories. It scans for test files and reports any missing specified suffixes, helping maintain consistency and adherence to project standards. The tool is not limited to any programming language files; it scans file names and ignores extensions until they are used in filters.
+
+## How It Works
+This action scans the specified `include_directories` for test files and checks if they end with the defined `suffixes` or contain them anywhere in the filename based on the `pattern_logic.` It reports the count of files not meeting the naming conventions, with options to fail the action if violations are found.
+
+## Inputs
+### `suffixes`
+- **Description**: List of suffixes that test files should have, separated by commas.
+- **Required**: Yes
+- **Default**: `UnitTests,IntegrationTests`
+
+### `include_directories`
+- **Description**: List of directories to include in the pattern check. This input limits scanning to these directories only.
+- **Required**: No
+- **Default**: `src/test/`
+
+### `exclude_directories`
+- **Description**: List of directories to exclude from scanning, separated by commas.
+- **Required**: No
+- **Default**: `dist,node_modules,coverage,target,.idea,.github,.git`
+
+### `exclude_files`
+- **Description**: List of filenames to exclude from suffix checks, separated by commas.
+- **Required**: No
+- **Default**: None
+
+### `case_sensitivity`
+- **Description**: Determines if the filename check should be case-sensitive.
+- **Required**: No
+- **Default**: `true`
+
+### `logic`
```

Comment:
`Logic off` or `Logic false` sounds misleading. I am still thinking of replacement word

Reply:
`suffix_logic` == True as default?

Reply:
Addressed in commit a52c31f6bfaea253b258aa435f51d01787a1ef9c.